requirements.txt

```diff
@@ -1,4 +1,4 @@
-attrs>=18.0,<20.4.0
+attrs>=18.0
```

Comment:
Since we need to safeguard against potential backward compatibility issues (or problematic versions), we'd rather limit the version here - we can limit to `,<22.1.0` which is the latest version at the moment and seems to be stable